### ISSUE TITLE
Intermittent timeout in /websockets/constructor/018.html

### ISSUE THREAD

**Issue by jdm** [servo/servo#10078]
```
  ▶ TIMEOUT [expected OK] /websockets/constructor/018.html

  ▶ Unexpected subtest result in /websockets/constructor/018.html:
  │ TIMEOUT [expected PASS] WebSockets: NULL char in url
  └   → Test timed out
```

**Comment by slayerjain** [comment]
@jdm  can i try this as my first bug?

**Comment by jdm** [comment]
This is not a good first bug, sorry. We've diagnosed the problem in one of the other issues about websocket timeouts, but don't know the solution yet.

**Comment by slayerjain** [comment]
then can you please help me find a good first bug?

**Comment by jdm** [comment]
should have been fixed by #10240.
